Resources/Prototypes/Entities/Clothing/Back/smuggler.yml

```diff
@@ -0,0 +1,1009 @@
+        prob: 0.01
+        orGroup: PresentsOrToys #3x small
+
+      - id: SpaceCash100
+        prob: 0.10
+        orGroup: Cash
+      - id: SpaceCash1000
+        prob: 0.10
+        maxAmount: 4
+        orGroup: Cash
+      - id: SpaceCash2500
+        prob: 0.05
+        maxAmount: 4
+        orGroup: Cash
+      - id: SpaceCash5000
+        prob: 0.01
+        maxAmount: 4
+        orGroup: Cash
+      - id: SpaceCash10000
+        prob: 0.005
+        maxAmount: 4
+        orGroup: Cash
+      - id: SpaceCash1000000
+        prob: 0.000001 #1:1,000,000 chance to win the lottery
+      - id: SpaceCash
+        prob: 0.01
+        orGroup: Cash #1x small
+
+      - id: SpaceCash100
+        prob: 0.9875
+        orGroup: Weapon
+      - id: Katana
+        prob: 0.0125
+        orGroup: Weapon
+      - id: ThrowingStar
+        prob: 0.0125
+        maxAmount: 5
+        orGroup: Weapon
+      - id: SpaceCash
+        prob: 0.0125
+        orGroup: Weapon  #5x small
+
+      - id: SpaceCash100
+        prob: 0.95
+        orGroup: Burger
+      - id: FoodBurgerAppendix
+        prob: 0.05
+        maxAmount: 5
+        orGroup: Burger
+      - id: FoodBurgerEmpowered
+        prob: 0.05
+        maxAmount: 5
+        orGroup: Burger
+      - id: FoodBurgerClown
+        prob: 0.05
+        maxAmount: 5
+        orGroup: Burger
+      - id: FoodBurgerGhost
+        prob: 0.05
+        maxAmount: 5
+        orGroup: Burger
+      - id: SpaceCash
+        prob: 0.05
+        orGroup: Burger #5x small
+
+      - id: SpaceCash100
+        prob: 0.95
+        orGroup: Generic2
+      - id: RemoteSignaller
+        prob: 0.05
+        orGroup: Generic2
+      - id: PersonalAI
+        prob: 0.05
+        orGroup: Generic2
+      - id: WeaponFlareGun
+        prob: 0.05
+        orGroup: Generic2
+      - id: ModularReceiver
+        prob: 0.05
+        orGroup: Generic2
+      - id: RifleStock
+        prob: 0.05
+        orGroup: Generic2
+      - id: SpaceCash
+        prob: 0.05
+        orGroup: Generic2  #1x small
+
+      - id: SpaceCash100
+        prob: 0.99
+        orGroup: Keys
+      - id: EncryptionKeyCommon
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyCargo
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyService
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyRobo
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyScience
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyMedical
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyEngineering
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeySyndie
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeySecurity
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyCommand
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyStationMaster
+        prob: 0.001
+        orGroup: Keys
```

Comment:
I think this ones kind of funny, but also seems like it could be a bad idea since these will mostly be found by non-antags.